modules/core/server/views/partials/header.server.view.html

```diff
@@ -1,270 +1,6 @@
-<header
-  id="tr-header"
-  role="banner"
-  class="navbar navbar-default navbar-fixed-top hidden-print"
+<app-header
+  is-i18n-enabled="app.appSettings.i18nEnabled"
```

Comment:
We can probably clean out the i18n feature flag out and thus no need to pass this config via props.

Reply:
PR for feature flag-removal: https://github.com/Trustroots/trustroots/pull/2217